gctscript/wrappers/gct/gctwrapper.go

```diff
@@ -0,0 +1,10 @@
+package gct
+
+import "github.com/thrasher-corp/gocryptotrader/gctscript/wrappers/gct/exchange"
+
+// Setup returns a Wrapper
+func Setup() *Wrapper {
+	return &Wrapper{
```

Comment:
return new(Wrapper)

Reply:
that wont initialise the embedded exchange struct at the same time